projects/valtimo/process-management/src/lib/process-management-builder/process-management-builder.component.ts

```diff
@@ -35,9 +36,12 @@ import CamundaModdleDescriptor from 'camunda-bpmn-moddle/resources/camunda.json'
 })
 export class ProcessManagementBuilderComponent implements OnInit, OnDestroy {
   public bpmnModeler;
+  public bpmnViewer;
   public processDefinitionVersions: ProcessDefinition[] | null = null;
   public selectedVersion: ProcessDefinition | null = null;
   public processKey: string | null = null;
+  public isReadOnlyProcess$: Observable<boolean> = of(false);
```

Comment:
This is not the way to use observables. It should be a BehaviourSubject of type boolean with initial value false